defaults/main.yml

```diff
@@ -0,0 +1,17 @@
+# Service accounts
+monitoring_sa: monitoring
+autobot_jenkins_sa: autobot-jenkins
+koji_sa: koji
+ansible_tower_sa: ansible-tower
+
+list_sa:
+  - "{{ monitoring_sa }}"
```

Comment:
if you want to have default list just add there something like 'example_sa' and that's it

Reply:
Fixed